### PR TITLE
Update performance issue title

### PR DESCRIPTION
**Pull request by sebastienros**

### CODE REVIEW

File: src/Microsoft.Crank.RegressionBot/Program.cs [hunk 1]
```diff
@@ -271,7 +271,7 @@ private static async Task CreateRegressionIssue(IEnumerable<Regression> regressi
 
             var body = await CreateIssueBody(regressions, template);            
             
-            var title = "Performance regression: " + String.Join(", ", regressions.Select(x => x.CurrentResult.Scenario).Take(5));
+            var title = "Performance variation: " + String.Join(", ", regressions.Select(x => x.CurrentResult.Scenario).Take(5));
```
Comment (davidfowl) [inline src/Microsoft.Crank.RegressionBot/Program.cs:274]: change?

Comment (sebastienros) [reply src/Microsoft.Crank.RegressionBot/Program.cs:274]: I like it, also deviation 

Other options: difference · disparity · imbalance · alteration · modification · variability · variance · divergence · fluctuation · modification

Comment (davidfowl) [reply src/Microsoft.Crank.RegressionBot/Program.cs:274]: difference

**Comment by sebastienros** [comment]
This will change the title. I will change the summary of the issue in the templates next. These are in the aspnet/benchmarks repository where the configuration for our scenarios is kept.

**Pull request merged**